NotificationHubs/src/com/windowsazure/messaging/Notification.java

```diff
@@ -139,7 +155,7 @@ public static Notification createAdmNotifiation(String body) {
 	 * @param body
 	 * @return
 	 */
-	public static Notification createBaiduNotifiation(String body) {
+	public static Notification createBaiduNotification(String body) {
```

Comment:
I don't think we need to make this a breaking change. Let's add the new methods with the correct names, and throw the `@deprecated` tag on the old methods with the misspelled names.

(I can't believe I'm saying that.)